sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/Configs.java

```diff
@@ -55,6 +55,10 @@
     private static final int CPU_CNT = Runtime.getRuntime().availableProcessors();
     private static final int DEFAULT_DIRECT_HTTPS_POOL_SIZE = CPU_CNT * 500;
 
+    //  Reactor Netty Constants
+    private static final int MAX_IDLE_CONNECTION_TIMEOUT_IN_MILLIS = 60 * 1000;
+    private static final int CONNECTION_ACQUIRE_TIMEOUT_IN_MILLIS = 45 * 1000;
```

Comment:
question: why do we have a different timeout value? any reason this one is 45s and the other is 60s? is there a rationale behind this?

Reply:
Yes, these are default values for reactor netty for these variables. I wanted to explicitly state them in our code. 
So that we can change them whenever we want.